packages/filecoin-rpc-client/src/index.ts

```diff
@@ -68,12 +54,9 @@ export default class LotusRpcEngine {
         jsonrpc: '2.0',
         method: `Filecoin.${method}`,
         params,
-        id: 1,
-      },
-      {
-        headers: this.headers,
-        ...this.axiosOpts,
+        id: 1
```

Comment:
Eventually i think it probably makes sense to allow an optional `id` param here so we can match responses with requests more easily (although this need hasn't come up yet). Or maybe there's another reason this id field exists on the JSON RPC spec. Don't know how we'd do this without breaking this method though... Probably should not be a part of this PR anyway, but just so it's on your radar..

Reply:
Hmm yeah good to know! Yeah maybe where there is a chance of performing multiple consequtive requests, maybe you just want to ignore the responses to the previous ones and only handle the one that matches the last `id`. Could help us in our `useEffect` handlers as well maybe. But yeah probably hard to change this while staying backwards compatible.

Reply:
Let's leave for now.